src/offline/mod_offline_chatmarkers_rdbms.erl

```diff
@@ -39,20 +63,13 @@ get(#jid{lserver = Host} = Jid) ->
 -spec maybe_store(Jid :: jid:jid(), Thread :: undefined | binary(),
                   Room :: undefined | jid:jid(), Timestamp :: integer()) -> ok.
 maybe_store(#jid{lserver = Host} = Jid, Thread, Room, Timestamp) ->
-    JidEscaped = escape(encode_jid(Jid)),
-    ThreadEscaped = escape(encode_thread(Thread)),
-    RoomEscaped = escape(encode_jid(Room)),
-    TSEscaped = escape(Timestamp),
-    InsertQuery = ["INSERT INTO offline_markers (jid, thread, room, timestamp) VALUES (",
-                   JidEscaped, ",", ThreadEscaped, ",", RoomEscaped, ",", TSEscaped, ");"],
-    Res = mongoose_rdbms:sql_query(Host, InsertQuery),
-    ok = check_insert_result(Res).
+    {updated, _} = execute_maybe_store(Host, encode_jid(Jid), encode_thread(Thread),
```

Comment:
Minor: in case of `execute_successfully` with a `UPDATE/DELETE` query, I see no reason to pattern match the result as it can be only `{updated, _}`. So I think we could simplify the code:

```suggestion
    execute_maybe_store(Host, encode_jid(Jid), encode_thread(Thread),
```